### PR TITLE
Accessibility fixes

### PR DESCRIPTION
**Pull request by keithjgrant**
##### SUMMARY

Fixes numerous accessibility issues, including:

* updates CodeEditor so label correctly points at associated textarea
* fixes issues with tabs on dashboard and details pages
* adds missings ids
* adds alt text to logo
* removes duplicate ids on some lists

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

File: awx/ui_next/src/components/AppContainer/AppContainer.jsx [hunk 1]
```diff
@@ -138,10 +138,12 @@ function AppContainer({ i18n, navRouteConfig = [], children }) {
     }
   }, [handleLogout, timeRemaining]);
 
+  const brandName = config?.license_info?.product_name || 'brand';
```
Comment (nixocio) [inline awx/ui_next/src/components/AppContainer/AppContainer.jsx]: `brand` was not selected to be translated. Is that the expected behavior?

File: awx/ui_next/src/components/AppContainer/BrandLogo.jsx [hunk 2]
```diff
@@ -13,6 +13,8 @@ const BrandImg = styled.img`
   pointer-events: none;
 `;
 
-const BrandLogo = () => <BrandImg src="/static/media/logo-header.svg" />;
+const BrandLogo = ({ alt }) => (
```
Comment (nixocio) [inline awx/ui_next/src/components/AppContainer/BrandLogo.jsx:16]: Do we need to provide an `alt` prop to all places that we are using `BrandLogo`? I just noticed the https://github.com/ansible/awx/blob/b8e05d6c6af3529284197e799d25dbfb6f1090bb/awx/ui_next/src/components/AppContainer/AppContainer.jsx#L161

File: awx/ui_next/src/components/AppContainer/AppContainer.jsx (hunk 1 above)
Comment (keithjgrant) [reply awx/ui_next/src/components/AppContainer/AppContainer.jsx]: oh good catch!

**Comment by tiagodread** [comment]
Running e2e tests

**Review by jakemcdermott — APPROVED** [review]

**Comment by keithjgrant** [comment]
Looks like this broke navigation via the tabs on details pages... working on a fix now

**Comment by keithjgrant** [comment]
> Looks like this broke navigation via the tabs on details pages... working on a fix now

I'm going to have to revert some of my changes here and circle back after PF adds support for custom link components: patternfly/patternfly-react#5659

**Review by tiagodread — APPROVED** [review]
- Fixed an e2e tests due to a locator change
- All e2e tests are 🟢 
- Moved from **46** violations to **5** **(given the current coverage existent)**

image

**Pull request merged**